pkg/svcgen/params/params.go

```diff
@@ -0,0 +1,96 @@
+package params
+
+type Params struct {
+	ProtoDir    string
+	ProtoOutDir string
+	ServerDir   string
+	Path        string
+	ServiceName string
+	Methods     []MethodParams
+	Proto       ProtoParams
+	PbGo        PbGoParams
+	Go          GoParams
+}
+
+type ProtoParams struct {
+	Package  string
+	Imports  []string
+	Messages []MethodMessage
+}
+
+type PbGoParams struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type PbGoParams should have comment or be unexported